xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -479,6 +479,11 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Category of vehicle.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="NumberOfWheels" type="xsd:nonNegativeInteger" minOccurs="0">
```

Comment:
```suggestion
			<xsd:element name="NumberOfWheels" type="xsd:positiveInteger" minOccurs="0">
```

Reply:
@Aurige Matthias provided a very sound example why it could be without wheels. So don't apply.